### PR TITLE
Bed respawn coords saves with tidy output

### PR DESCRIPTION
**Pull request by tenplus1**
Tweaked the beds.save_spawns() function to compile list then output to file with shortened coords containing 1 decimal place.

### CODE REVIEW

File: mods/beds/spawns.lua [hunk 1]
```diff
@@ -38,13 +38,13 @@ function beds.read_spawns()
 end
 
 function beds.save_spawns()
-	if not beds.spawn then
-		return
-	end
+	if not beds.spawn then return end
```
Comment (PilzAdam) [inline mods/beds/spawns.lua]: I don't like these inline ifs.

File: mods/beds/spawns.lua [hunk 2]
```diff
@@ -41,10 +41,12 @@ function beds.save_spawns()
 	if not beds.spawn then
 		return
 	end
+	local data = {}
 	local output = io.open(org_file, "w")
 	for i, v in pairs(beds.spawn) do
```
Comment (Rui914) [inline mods/beds/spawns.lua]: for k, v

Comment (tenplus1) [inline mods/beds/spawns.lua]: explain your comment please.

Comment (Rui914) [inline mods/beds/spawns.lua]: I think you should change this too.
```
for i, v
```
to
```
for k, v
```

Comment (tenplus1) [inline mods/beds/spawns.lua]: Right...  Glad you cleared that up, but for the sake of a variable name I'll leave it as is so long as things are working.

**Comment by SmallJoker** [comment]
I'm not sure if a `v.y + 1` (for feet offset) and a following `math.round(v)` could do it since the beds have a fixed node position.

**Comment by paramat** [comment]
:+1: If tested.

**Comment by tenplus1** [comment]
Have been using this change on Xanadu server for over a week and it's working fine.

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
Approved by nore http://irc.minetest.ru/minetest-dev/2016-04-29#i_4597385

**Comment by paramat** [comment]
dc7cbdd5ab9efa83327f057a4e4376ffacab6c0e

**Pull request closed without merging**